### DISCUSSION TITLE
Migrate Flutter V2

### DISCUSSION THREAD

**Discussion by yamisagi** [DetainedDeveloper/App-Brewery-Flutter-Null-Safety#3] [Imrpovements]
**All we know Course materials are out-dated**

To **Fix** this issue easily ,
Delete all files and folders in App folder **except** couse materials **lib folder-Assets-Fonts-pubspec.yaml**

For **_Example_**;

![1](https://user-images.githubusercontent.com/84624853/152107377-fe39a72f-8386-4e51-89a8-8a1466dec4b8.jpg)

And then go to **Terminal** while **its in project folder** ,
Write this line to **Terminal**
`flutter create .`

For _**Example**_;

![2](https://user-images.githubusercontent.com/84624853/152107399-196a6774-5b8d-4ccb-9407-311a490b24fa.jpg)

And then Flutter starts rebuilding application with migrated version of it.

Hope this works.

**Comment by MiladZarour** [comment]
That was really helpful!
Thx 🙏🙏
